Formula/notcurses.rb

```diff
@@ -0,0 +1,32 @@
+class Notcurses < Formula
+  desc "Blingful character graphics/TUI library"
+  homepage "https://nick-black.com/dankwiki/index.php/Notcurses"
+  url "https://github.com/dankamongmen/notcurses/archive/refs/tags/v2.3.12.tar.gz"
+  sha256 "ce042908fac11f7df1f9eaa610e46e9c615f53ab036b7c27ae2396292512407b"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "doctest" => :build
+  depends_on "pandoc" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ffmpeg"
+  depends_on "libunistring"
```

Comment:
We missed the Mojave failure, with the error
```
/tmp/notcurses-20210729-56842-1cecch7/notcurses-2.3.12/src/ls/main.cpp:37:20: error: 'path' is unavailable: introduced in macOS 10.15
/Applications/Xcode.app/Contents/Developer/usr/bin/make  -f CMakeFiles/widestomp.dir/build.make CMakeFiles/widestomp.dir/depend
  std::filesystem::path dir;
                   ^
```
I don't think there's a way around that, so
```suggestion
  depends_on "libunistring"
  depends_on macos: :catalina # requires std::filesystem
```

Reply:
i'll resolve this for Mojave upstream, and remove this restriction on the next bump. i've gone ahead and filed https://github.com/dankamongmen/notcurses/issues/2001 about this.